libs/api/domains/license-service/src/lib/client/driving-license-client/drivingLicenseService.api.ts

```diff
@@ -158,9 +163,51 @@ export class GenericDrivingLicenseApi
     return licenses
   }
 
+  private async getCachedGetPkPassToken(): Promise<string | null> {
+    if (!this.cacheManager) {
+      return null
+    }
+
+    const cached = await this.cacheManager?.get(CACHE_KEY)
+
+    if (cached && typeof cached === 'string' && cached.length > 0) {
```

Comment:
```suggestion
    if (cached && typeof cached === 'string') {
```

The length check is redundant since cached is known to be `true`y